test/server/cards/01_SOR/units/MiningGuildTieFighter.spec.ts

```diff
@@ -0,0 +1,38 @@
+describe('Mining Guild TIE Fighter', function() {
+    integration(function(contextRef) {
+        describe('Mining Guild TIE Fighter\'s ability', function() {
+            beforeEach(function () {
```

Comment:
Not sure if you saw but Gart found a way to put the test setup directly in the test case if there is only one test that uses this setup. Not required to use it if you prefer to do it this way, just an fyi in case you like the idea. See Reinforcement Walker tests for an example.

Reply:
Ah yeah I see below you use this in other places